packages/playground-examples/copy/pt/4-0/New TS Features/Named Tuples.ts

```diff
@@ -0,0 +1,51 @@
+//// { compiler: { ts: "4.0.0-beta" } }
+// Tuplas são vetores que sua ordem é importante para a tipagem,
+// você pode aprender mais sobre eles no exemplo:tuplas
```

Comment:
```suggestion
// Tuplas são vetores que sua ordem é importante para a tipagem,
// você pode aprender mais sobre elas no exemplo: example:tuples
```

Sintaxe `example:*` são links para outros exemplos